Orbitersdk/samples/ProjectApollo/src_launch/mcc.cpp

```diff
@@ -2412,11 +2412,66 @@ void MCC::drawPad(bool writetofile){
 		{
 			int hh, mm;
 			double ss;
+			char buffer2[1024];
```

Comment:
These names are not descriptive.
Looking at your code buffer2 is some kind of intermediate that you use to construct the current line and buffer 3 is the string of the complete PAD.

Reply:
See, you figured it out easily. Descriptive enough. If you suggest some names I'll change it, but I don't think it's super necessary.

Reply:
That is not really what I meant...
I was hoping my explanation of what I think you are trying was enough of a hint for a proper name.

In order to understand what `buffer2` means I have to read a whole section to understand its purpose.
If they would've been named something like `padLine` or `currentLine` I wouldn't need to spend 20 seconds reading ahead just to understand what you are even trying to do.

Reply:
Well "line" is more confusing as even the updated code doesn't handle the PAD line by line. I will use tempString and fullString, I think that gets across what it does enough.